### DISCUSSION TITLE
<title>page opening issue

### DISCUSSION THREAD

**Discussion by Thomas-Dhanabal-I** [community/community#38919] [Projects and Issues]
### Select Topic Area

General

### Body

if i open any assignments which done by others,it shows like this
![Screenshot (55)](https://user-images.githubusercontent.com/118050288/201469944-f7eff12f-3913-469d-a186-71ceef5e4620.png)

@airtower-luna
@queenofcorgis
@zackfern

**Comment by jfullstackdev** [comment]
that load fine on my end 

![image](https://user-images.githubusercontent.com/47092464/201529440-e102eda3-a8ab-48e0-b072-1d2451e121bc.png)

seems that is not GitHub issue

if you have any extensions installed, that can affect, try Incognito if it will load, 

if it is not still working in Incognito mode, then check the RAM of your computer, slow Internet connection, antivirus, firewall etc.,
